### ISSUE TITLE
Figure out why it doesn't work on c3 HDD

### ISSUE THREAD

**Issue by mvisonneau** [mvisonneau/automount#7]
Apparently the ghw.Block() function doesn't seem to return the drives correctly on c3 instances (AWS) :

```
~$ automount --log-level debug mount /mnt
INFO[2019-05-29T16:09:36Z] No device specified, trying to find one automatically
INFO[2019-05-29T16:09:36Z] Found 0 disk(s), total size of 0 GB
WARN[2019-05-29T16:09:36Z] No available device found, exiting
DEBU[2019-05-29T16:09:36Z] Executed in 317.579µs, exiting..

~$ lsblk
NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
xvda    202:0    0   10G  0 disk
└─xvda1 202:1    0   10G  0 part /
xvdb    202:16   0  320G  0 disk /mnt
xvdc    202:32   0  320G  0 disk
```

**Comment by mvisonneau** [comment]
This has been sorted out in https://github.com/jaypipes/ghw/pull/130 🎉

**Comment by mvisonneau** [comment]
released in **0.1.0**
